### PR TITLE
YG-149 Create API to download Sample CSV file for the selected template

### PR DESCRIPTION
**Pull request by pepcus-gajendra**
# Related issues

Create API to download Sample CSV file for the selected template

### CODE REVIEW

**Review by pepcus-mukatip — APPROVED** [review]
approved

**Review by pepcus-ankit — CHANGES_REQUESTED** [review]

File: src/main/java/com/pepcus/appstudent/service/TemplateService.java [hunk 1]
```diff
@@ -1,22 +1,50 @@
...
+			// String filepath = "C:\\Users\\Gajendra Thakur\\Desktop\\csv\\";
+			File file = new File(filePath + template.getTemplateName() +ApplicationConstants.CSV_EXTENSION);
+			InputStream inputStream = new FileInputStream(file);
+			response.setContentType("application/force-download");
+			response.setHeader("Content-Disposition", "attachment; file=" + file.getName());
+			IOUtils.copy(inputStream, response.getOutputStream());
+			response.flushBuffer();
+			inputStream.close();
+		} catch (Exception exception) {
+			System.out.println(exception.getMessage());
```
Comment (pepcus-ankit) [inline src/main/java/com/pepcus/appstudent/service/TemplateService.java:46]: We should not commit System.out.println. 
@pepcus-gajendra  @pepcus-mukatip  Please review carefully.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -30,4 +30,5 @@ send_reg_sms_gms_student=true
 send_payment_sms_gms_student=true
 gms_reg_payment_amt=100
 gms_payment_contact_number=09893951912
-admin.Id=123
\ No newline at end of file
+admin.Id=123
+download.csv.file.path=/resource/
```
Comment (pepcus-ankit) [inline src/main/resources/application.properties:34]: Can we rename this property to more appropriate name like.
csv.template.path

**Review by pepcus-ankit — CHANGES_REQUESTED** [review]

File: src/main/resources/YG-147.SQL [hunk 3]
```diff
@@ -1,4 +1,5 @@
-ALTER TABLE .`student` 
+DELIMITER $$
```
Comment (pepcus-mukatip) [inline src/main/resources/YG-147.SQL:1]: why delimiter used with Alter statement?

**Review by pepcus-ankit — APPROVED** [review]

**Pull request merged**